tests/functional/deployment/test_deployment.py

```diff
@@ -56,6 +59,11 @@ def test_deployment(pvc_factory, pod_factory):
                 if config.ENV_DATA["mcg_only_deployment"]:
                     mcg_only_install_verification(ocs_registry_image=ocs_registry_image)
                     return
+                elif config.ENV_DATA["odf_provider_mode_deployment"]:
```

Comment:
Since the `odf_provider_mode_deployment` parameter is not configured in default config, you should probably use the `get(...)` method similarly as above.